binding/check.go

```diff
@@ -0,0 +1,33 @@
+package binding
+
+import (
+	"fyne.io/fyne/widget"
+)
+
+func BindCheckChanged(check *widget.Check, data *BoolBinding) {
+	existing := check.OnChanged
```

Comment:
Where the "main state" is different to something else (like label below) maybe one of them is primary so BindCheck() is the one associated with the main Value? having "Changed" on the end seems like it's binding to an event.

Reply:
I guess I'm not sure of the intent for this method, so it's tough to suggest better names. I assume you are not literally binding to OnChanged as that is an event, not a state. Can you provide some context for this?